### PR TITLE
feat: add LOCAL_CHAIN_BLOCK_MINING_INTERVAL and note useful envs

### PR DESCRIPTION
**Pull request by AlcibiadesCleinias**
* add LOCAL_CHAIN_BLOCK_MINING_INTERVAL
* more envs to show

### CODE REVIEW

File: .example.env [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Private key to use for sender of transactions (deploy, etc) in forge scripts for {testnet, kras, stage}.
```
Comment (folex) [inline .example.env:1]: Will these vars be configurable via env in Docker container?

File: .example.env [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+PRIVATE_KEY=0x0
+# Wait confirmation in scripts and tests.
+WAIT_CONFIRMATIONS=5
+# Local Chain (Anvil Node).
+LOCAL_CHAIN_BLOCK_MINING_INTERVAL=15
```
Comment (folex) [inline .example.env:6]: Let's set to 1 by default maybe? Who needs 15 seconds on local chain?

File: .example.env (hunk 1 above)
Comment (AlcibiadesCleinias) [reply .example.env:1]: it relates to deploy scripts. If envs set in docker and then makefile commands run in docker, values from env will be used

File: .example.env (hunk 2 above)
Comment (AlcibiadesCleinias) [reply .example.env:6]: we have right now 15 everywhere, need to check first of all the setup for 1 sec... I am too afraid and do not want to debug it right now in the case

File: .example.env (hunk 1 above)
Comment (folex) [reply .example.env:1]: https://github.com/fluencelabs/deal/blob/main/docker/Dockerfile#L48-L51

Seems like it does.

**Review by folex — APPROVED** [review]

File: .example.env (hunk 1 above)
Comment (AlcibiadesCleinias) [reply .example.env:1]: so, does that mean that that is okie and as expected?

Comment (folex) [reply .example.env:1]: Seems that it will be available to configure via envs to docker container, yes.

File: .example.env (hunk 2 above)
Comment (folex) [reply .example.env:6]: Okay

**Pull request merged**